src/Symfony/Component/Mailer/Bridge/Amazon/Transport/SesApiTransport.php

```diff
@@ -62,12 +63,13 @@ protected function doSendApi(Email $email, Envelope $envelope): ResponseInterfac
             'body' => $this->getPayload($email, $envelope),
         ]);
 
+        $result = new \SimpleXMLElement($response->getContent(false));
         if (200 !== $response->getStatusCode()) {
-            $error = new \SimpleXMLElement($response->getContent(false));
-
-            throw new HttpTransportException(sprintf('Unable to send an email: %s (code %s).', $error->Error->Message, $error->Error->Code), $response);
+            throw new HttpTransportException(sprintf('Unable to send an email: %s (code %s).', $result->Error->Message, $result->Error->Code), $response);
         }
 
+        $sentMessage->setMessageId($result->SendEmailResult->MessageId);
```

Comment:
Just for reference: #33326  / @connorhu 
(In case using `aws-sdk` use would affect direct XML parsing versus `$result->get('MessageId')`)

Reply:
But we're not using the SDK.